### ISSUE TITLE
Less Spammy Telegram

### ISSUE THREAD

**Issue by rcarmo** [whittlem/pycryptobot#323]
**Is your feature request related to a problem? Please describe.**
Please disable candlestick detection messages in Telegram, since they can be quite spammy.

**Describe the solution you'd like**
A flag to mute them, set to default

**Describe alternatives you've considered**
Hacking that out of the code

**Comment by emreeroglu** [comment]
This will be configurable when Telegram integrated with logger. So user will be able to pick which level of logs to get Telegram.
Telegram (in general Chat) logs may be have a separate log level than logger so chat configuration would have a little more detail.

**Comment by whittlem** [comment]
I agree with @rcarmo that candlestick detection messages are spammy as a default. As a default Telegram should only be receiving buy and sell events. @emreeroglu if the user wants to receive more verbose messaging then they could increase the logging level as you describe. Are you able to help me remove those spammy messages as a default?

**Comment by emreeroglu** [comment]
I would like to help, have some design ideas but need to finalize those and discuss with you so we can be future proof about possible new chat clients.

**Comment by hebawom** [comment]
I was at a wedding the other day and my watch kept pinging me candlesticks!

**Comment by devinat1** [comment]
I am writing this comment in agreement that it would be nice to customize which types of messages to receive from telegram (via the JSON file for example).

**Comment by Nicolas469** [comment]
Any progress here ? I think that we have two options: one flag to disable candlestick detection or add a notification level.

**Comment by loomsen** [comment]
I was thinking about implementing telegram (and possibly other messengers) as logging handlers in python. Did a quick proof of concept:

![image](https://user-images.githubusercontent.com/302524/129233950-73f08bda-c893-4c6b-ab94-4680dc0cad76.png)

Leads to:
![image](https://user-images.githubusercontent.com/302524/129234014-6dcc1aab-a298-4def-9385-6e71aa9fc8a4.png)


So we could just use `log.debug`, `log.error`, `log.info` etc and let the handlers handle it. Didn't have the time to implement, but it's not exactly rocket science, so if someone wants to...
